### PR TITLE
Update tidb-roadmap.md since last LTS

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13615
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-roadmap.md [hunk 1]
```diff
@@ -26,56 +28,33 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
       <td>
         <ul>
           <li>
-            <b>通用执行计划缓存</b><br />提升通用的读性能
-          </li>
-          <br />
-          <li>
-            <b>分区 Raft KV 存储引擎</b><br />提供更快的写入速度、更快的扩缩容操作，支持更大的集群
+            <b>分区 Raft KV 存储引擎 GA</b><br />支持 PB 级别集群，提升写入速度、扩缩容操作速度，改进 Compaction 稳定性
```
Comment (ran-huang) [inline tidb-roadmap.md:31]:
```suggestion
            <b>分区 Raft KV 存储引擎 GA</b><br />支持 PB 级别的集群，提升写入速度、扩缩容操作速度，提升数据整理的稳定性
```

File: tidb-roadmap.md [hunk 2]
```diff
@@ -26,56 +28,33 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
           <li>
-            <b>PB 级别的稳定性</b><br />
-            为海量数据提供可靠且稳定的性能
-          </li>
-          <br />
-          <li>
-            <b>TiFlash 计算和存储分离架构 （自动缩放）</b><br />
-            实现弹性利用 HTAP 资源
+            <b>引入性能优化框架，适用于所有相关后台任务，如 DDL、TTL 和集群分析操作</b><br />
+            性能优化框架将这些操作的工作负载分布到整个集群中，从而加快了性能，并减少了各个节点上的资源消耗。该框架已经应用于 <code>ADD INDEX</code> 操作。
```
Comment (ran-huang) [inline tidb-roadmap.md:44]:
```suggestion
            性能优化框架将这些后台任务的工作负载分散到整个集群中，从而提升性能，并减少各个节点上的资源消耗。该框架已经应用于 <code>ADD INDEX</code> 操作。
```

File: tidb-roadmap.md [hunk 3]
```diff
@@ -26,56 +28,33 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
         </ul>
       </td>
       <td>
         <ul>
-          <li>
-            <b>更强大的下一代存储引擎</b>
-          </li>
-          <br />
           <li>
             <b>事务大小无限制</b>
```
Comment (ran-huang) [inline tidb-roadmap.md:56]:
```suggestion
            <b>移除事务大小的限制</b>
```

File: tidb-roadmap.md [hunk 4]
```diff
@@ -87,32 +66,16 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
-          <li>
-            <b>增强 TiCDC 和 PITR 的恢复目标</b>
-            <br />增加业务连续性，将系统故障的影响最小化
-          </li>
-          <br />
-          <li>
-            <b>TiProxy</b>
-            <br />在集群升级、扩缩容时，保持应用与数据库的连接，避免影响业务
+            <b>后台任务支持资源控制</b><br />
+            控制后台任务（如数据导入、DDL、TTL、自动分析、数据整理等）对前台流量的影响
```
Comment (ran-huang) [inline tidb-roadmap.md:70]:
```suggestion
            控制后台任务（如数据导入、DDL、TTL、自动分析、数据整理等操作）对前台流量的影响
```

File: tidb-roadmap.md [hunk 5]
```diff
@@ -263,45 +179,45 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
-            <br />提供安全和标准的身份验证方式
+            <b>通过 Azure Key Vault 进行密钥管理</b>
+            <br />由 Azure Key Vault 管理的静态加密
           </li>
           <br />
           <li>
-            <b>LDAP 集成</b>
-            <br />通过 TLS 在 LDAP 服务器进行身份验证
+            <b>列级访问控制</b>
+            <br />授予和限制对特定列的访问权限
```
Comment (ran-huang) [inline tidb-roadmap.md:193]:
```suggestion
            <br />允许针对特定列来授予或限制访问权限
```

File: tidb-roadmap.md [hunk 6]
```diff
@@ -263,45 +179,45 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
           </li>
           <br />
           <li>
-            <b>增强审计日志</b>
-            <br />
-            审计日志中提供更多细节
+            <b>数据库级别的加密</b>
+            <br />在数据库级别配置的静态加密
```
Comment (ran-huang) [inline tidb-roadmap.md:198]:
```suggestion
            <br />支持配置数据库级别的静态加密
```

File: tidb-roadmap.md [hunk 7]
```diff
@@ -322,7 +237,22 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
+
+历史版本的路线图中的部分功能已交付，列举如下。更多详细信息，请参阅 [v7.1.0 Release Notes](/releases/release-7.1.0.md)。
+
+- 多租户框架的基础：资源组的资源控制配额和调度
+- TiCDC 支持对象存储 sink，包括 Amazon S3 和 Azure Blob Storage (GA)
+- 最快的在线添加索引 `ADD INDEX` 操作 (GA)
+- TiFlash 延迟物化 (GA)
+- TiFlash 支持数据落盘 (GA)
+- LDAP 身份验证
+- 增强的 SQL 审计日志（仅企业版可用）
```
Comment (ran-huang) [inline tidb-roadmap.md:250]:
```suggestion
- SQL 审计日志增强（仅企业版可用）
```

File: tidb-roadmap.md [hunk 8]
```diff
@@ -322,7 +237,22 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
+- 分区 Raft KV 存储引擎（实验性）
```
Comment (ran-huang) [inline tidb-roadmap.md:251]:
```suggestion
- 分区 Raft KV 存储引擎（实验特性）
```

File: tidb-roadmap.md [hunk 9]
```diff
@@ -322,7 +237,22 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
+- 通用的 Session 级别执行计划缓存（实验特性）
```
Comment (ran-huang) [inline tidb-roadmap.md:252]:
```suggestion
- 通用的会话级别执行计划缓存（实验特性）
```

File: tidb-roadmap.md [hunk 10]
```diff
@@ -322,7 +237,22 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
+- TiCDC 支持以 Kafka 为下游的分布式表级别数据同步（实验性）
```
Comment (ran-huang) [inline tidb-roadmap.md:253]:
```suggestion
- TiCDC 支持以 Kafka 为下游的分布式表级别数据同步（实验特性）
```

File: tidb-roadmap.md [hunk 11]
```diff
@@ -263,45 +179,45 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
           </li>
           <br />
           <li>
-            <b>LDAP 集成</b>
-            <br />通过 TLS 在 LDAP 服务器进行身份验证
+            <b>列级访问控制</b>
+            <br />允许针对特定列来授予或限制访问权限
           </li>
           <br />
           <li>
```
Comment (ran-huang) [inline tidb-roadmap.md:196]:
```suggestion
```

File: tidb-roadmap.md [hunk 12]
```diff
@@ -263,45 +179,45 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
-            <b>增强审计日志</b>
-            <br />
-            审计日志中提供更多细节
+            <b>数据库级别的加密</b>
+            <br />支持配置数据库级别的静态加密
```
Comment (ran-huang) [inline tidb-roadmap.md:198]: 上一版本更新时，中文版移除了这一条，原因是“仅在企业版中发布，不适合写在 roadmap 中”
```suggestion
```

File: tidb-roadmap.md
Comment (ran-huang) [inline tidb-roadmap.md:1]: Preview: https://preview-pingcap-docs-cn-1440.pingcap-docs-website-scaffold-1hq.pages.dev/zh/tidb/dev/tidb-roadmap

File: tidb-roadmap.md [hunk 13]
```diff
@@ -87,56 +66,24 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
       <td>
         <ul>
           <li>
-            <b>资源控制：资源组和后台任务支持流控和优先级调度</b><br />
-            稳定高效地管理共享集群的负载和业务
+            <b>后台任务支持资源控制</b><br />
```
Comment (Oreoxmt) [inline tidb-roadmap.md:69]:
```suggestion
            <b>后台任务支持资源管控</b><br />
```
https://docs.pingcap.com/zh/tidb/dev/tidb-resource-control

File: tidb-roadmap.md [hunk 14]
```diff
@@ -87,56 +66,24 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
         <ul>
           <li>
             <b>多租户</b>
-            <br />提供细粒度的资源控制和隔离，降低成本
-          </li>
-          <br />
-          <li>
-            <b>提升集群级和节点级的容错能力</b>
-            <br />增强集群的弹性
+            <br />基于资源控制实现资源隔离
```
Comment (Oreoxmt) [inline tidb-roadmap.md:78]:
```suggestion
            <br />基于资源管控实现资源隔离
```

File: tidb-roadmap.md [hunk 15]
```diff
@@ -322,7 +237,22 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
 
 上述表格中并未列出所有功能，当前规划可能会调整。不同的服务订阅版本中的功能可能有所不同。
 
-## 近期已发布特性
+## 已发布特性
+
+历史版本的路线图中的部分功能已交付，列举如下。更多详细信息，请参阅 [v7.1.0 Release Notes](/releases/release-7.1.0.md)。
+
+- 多租户框架的基础：资源组的资源控制配额和调度
```
Comment (Oreoxmt) [inline tidb-roadmap.md:244]:
```suggestion
- 多租户框架的基础：资源组的资源管控配额和调度
```

File: tidb-roadmap.md [hunk 16]
```diff
@@ -322,7 +237,22 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
+- TiCDC 支持对象存储 sink，包括 Amazon S3 和 Azure Blob Storage (GA)
+- 最快的在线添加索引 `ADD INDEX` 操作 (GA)
+- TiFlash 延迟物化 (GA)
+- TiFlash 支持数据落盘 (GA)
+- LDAP 身份验证
```
Comment (Oreoxmt) [inline tidb-roadmap.md:249]:
```suggestion
- LDAP 身份认证
```

File: tidb-roadmap.md [hunk 17]
```diff
@@ -322,7 +237,22 @@ TiDB 路线图展示了 TiDB 未来的发展方向，包括新特性和改进提
...
+- SQL 审计日志增强（仅企业版可用）
```
Comment (Oreoxmt) [inline tidb-roadmap.md:250]: @ran-huang 这条仅企业版可用的特性可以写在这里吗

Comment (ran-huang) [reply tidb-roadmap.md:250]: 确认过了和海外保持一致

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**